Common/src/Storage/TaskLifeCycleHelper.cs

```diff
@@ -246,10 +246,19 @@ await resultTable.ChangeResultOwnership(parentTaskId,
                        .ConfigureAwait(false);
     }
 
+    var readyTask = await prepareTaskDependencies.ConfigureAwait(false);
+    await taskRequests.Select(request => request.TaskId)
+                      .Chunk(100)
+                      .ParallelForEach(strings => taskTable.UpdateManyTasks(data => data.Status == TaskStatus.Creating && strings.Contains(data.TaskId),
+                                                                            new UpdateDefinition<TaskData>().Set(data => data.Status,
+                                                                                                                 TaskStatus.Pending),
+                                                                            cancellationToken))
+                      .ConfigureAwait(false);
```

Comment:
We did not chunk those kind of requests in the rest of the code. I am not sure of the impact, though.

Reply:
I'll remove the chunk for consistency purpose. To test further